server/common/nosql/mongo-dao/src/test/java/org/kaaproject/kaa/server/common/nosql/mongo/dao/CredentialsMongoDaoTest.java

```diff
@@ -38,9 +43,42 @@ public void afterTest() throws IOException {
     }
 
     @Test
-    public void findCredentialByIdTest() {
-        String id = "56f3f9acdd796a86323f04a4";
-        Credentials credentials = credentialsDao.findById(id);
-        Assert.notNull(credentials);
+    public void testSaveCredentials() {
+        CredentialsDto credentials = this.generateCredentials(CREDENTIALS_BODY, AVAILABLE);
+        Credentials saved = this.credentialsDao.save(credentials);
```

Comment:
The credentials has been already saved to the database by the `AbstractTest#generateCredentials(byte[], CredentialsStatus)` method.